pkg/kubelet/apis/config/types.go

```diff
@@ -100,6 +100,8 @@ type KubeletConfiguration struct {
 	// volumePluginDir is the full path of the directory in which to search
 	// for additional third party volume plugins.
 	VolumePluginDir string
+	// cloudProvider is the provider for cloud services.
```

Comment:
copy the flag documentation here (meaning of empty string, etc), and maybe also call out the special "external" value?